src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -111,48 +116,20 @@ internal Conversions Conversions
         /// </summary>
         private HashSet<SyntaxTree> _lazyCompilationUnitCompletedTrees;
 
-        public override string Language
-        {
-            get
-            {
-                return LanguageNames.CSharp;
-            }
-        }
+        public override string Language => LanguageNames.CSharp;
 
-        public override bool IsCaseSensitive
-        {
-            get
-            {
-                return true;
-            }
-        }
+        public override bool IsCaseSensitive => true;
 
         /// <summary>
         /// The options the compilation was created with. 
         /// </summary>
-        public new CSharpCompilationOptions Options
-        {
-            get
-            {
-                return _options;
-            }
-        }
+        public new CSharpCompilationOptions Options => _options;
 
-        internal AnonymousTypeManager AnonymousTypeManager
-        {
-            get
-            {
-                return _anonymousTypeManager;
-            }
-        }
+        internal AnonymousTypeManager AnonymousTypeManager => _anonymousTypeManager;
 
-        internal override CommonAnonymousTypeManager CommonAnonymousTypeManager
-        {
-            get
-            {
-                return AnonymousTypeManager;
-            }
-        }
+        internal override CommonAnonymousTypeManager CommonAnonymousTypeManager => AnonymousTypeManager;
+
+        internal DelegateCacheManager DelegateCacheManager => _delegateCacheManager;
```

Comment:
I forgot putting this as a question before calling for review, good catch. Will move, thanks.